packages/config/config/devices/0x0129/ntm625.json

```
@@ -322,37 +321,57 @@
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"23": {
-			"label": "Lock Body Alarms Mask",
-			"description": "Each bit controls one alarm. 1 = alarm enabled, 0 = alarm disabled. 0xFF = All Alarms on, Default is 0xFF. Bit 0: DPS Alarm (AX) (Duplicate of parameter 19. Values should be tied together) Bit 1: Deadbolt Alarm Bit 2: Lever Rotated (Rx) Bit 3 – 7: Not Used",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 255,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "DPS Alarm (AX) (Duplicate of parameter 19. Values should be tied together)",
+					"label": "Off",
 					"value": 0
 				},
 				{
-					"label": "Deadbolt Alarm",
-					"value": 1
-				},
-				{
-					"label": "Lever Rotated",
-					"value": 2
-				},
-				{
-					"label": "Off",
+					"label": "On",
 					"value": 255
 				}
 			]
+		},
+		"23[0x00]": {
```

```suggestion
		"23[0x01]": {
```
Bit 0 is 2^0 = 1, Bit 1 is 2^1 = 2, Bit 2 = 2^2 = 4, and so on